cognite/client/_api/files.py

```diff
@@ -662,7 +664,7 @@ def upload_content_bytes(
         return file_metadata
 
     def _upload_bytes(self, content: bytes | TextIO | BinaryIO, res: Response) -> FileMetadata:
-        returned_file_metadata = res.json()
+        returned_file_metadata = res.json()["items"][0]
```

Comment:
Ah, this is now used for both the old and new upload strategy, so I guess we'll need to pass in the actual upload_url rather than the raw response.